### ISSUE TITLE
About confidence

### ISSUE THREAD

**Issue by WashingtonTsai** [M2219/ESMStereo#4]
Thank you for such great work and open-source spirit. May I ask how the ground truth for confidence is generated and trained?

**Comment by M2219** [comment]
@WashingtonTsai, look at https://github.com/seungryong/LAF for more information on confidence map. Confidence ground truth is the errors less than 3 pixels between the disparity ground truth and the estimated disparity fed to a  binary cross entropy loss. 

gt_conf = (torch.abs(disp_ests[0] - disp_gts[0]) <= 3).type(dtype=torch.float)
